### PR TITLE
Refactor msr

### LINKED ISSUE #592: Remove "MSR"
Because the term `MSR` (measurement) is not generally used in the literature, I propose modifying the protocol names from "protocol name_msr" to "protocol name_signal." This adjustment should be reflected in the names of the protocols files and in the report by changing the axis labels and units following #275.

### PR DESCRIPTION
**Pull request by GabrielePalazzo**
Closes #592. Renamed MSR to Signal.
~~Units are still in uV.~~

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

**Comment by andrea-pasquale** [comment]
> Units are still in uV.

At this point I suggest also to drop uV and keep what it is returned by the instruments given that we play with arbitrary units.

File: src/qibocal/protocols/characterization/coherence/t1_signal.py [hunk 1]
```diff
@@ -88,8 +88,8 @@ def _acquisition(
         type=SweeperType.ABSOLUTE,
     )
 
-    # create a DataUnits object to store the MSR, phase, i, q and the delay time
-    data = T1MSRData()
+    # create a DataUnits object to store the Signal, phase, i, q and the delay time
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/t1_signal.py:91]:
```suggestion
```

File: src/qibocal/protocols/characterization/coherence/t1_signal.py [hunk 2]
```diff
@@ -174,13 +174,13 @@ def _plot(data: T1MSRData, qubit, fit: T1MSRResults = None):
         showlegend=True,
         uirevision="0",  # ``uirevision`` allows zooming while live plotting
         xaxis_title="Time (ns)",
-        yaxis_title="MSR (uV)",
+        yaxis_title="Signal",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/t1_signal.py:177]: Maybe "Signal (a.u.)" is better, what do you think @andrea-pasquale ?

File: src/qibocal/protocols/characterization/coherence/t2_sequences.py [hunk 3]
```diff
@@ -43,9 +43,9 @@ def _acquisition(
     )
 
     # create a DataUnits object to store the results,
-    # DataUnits stores by default MSR, phase, i, q
+    # DataUnits stores by default Signal, phase, i, q
     # additionally include wait time and t_max
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/t2_sequences.py:47]: Delete

File: src/qibocal/protocols/characterization/flipping.py [hunk 4]
```diff
@@ -75,7 +73,7 @@ def _acquisition(
         data (:class:`FlippingData`)
     """
 
-    # create a DataUnits object to store MSR, phase, i, q and the number of flips
+    # create a DataUnits object to store Signal, phase, i, q and the number of flips
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/flipping.py:76]:
```suggestion
```

File: src/qibocal/protocols/characterization/ramsey_sequences.py [hunk 5]
```diff
@@ -44,7 +44,7 @@ def _acquisition(
     )
 
     # create a DataUnits object to store the results,
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/ramsey_sequences.py:46]: Delete

File: src/qibocal/protocols/characterization/resonator_punchout.py [hunk 6]
```diff
@@ -135,7 +127,7 @@ def _acquisition(
     )
 
     # create a DataUnits object to store the results,
-    # DataUnits stores by default MSR, phase, i, q
+    # DataUnits stores by default Signal, phase, i, q
     # additionally include resonator frequency and attenuation
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/resonator_punchout.py:131]: Delete

File: src/qibocal/protocols/characterization/coherence/t2_signal.py [hunk 7]
```diff
@@ -63,9 +63,9 @@ def _acquisition(
     )
 
     # create a DataUnits object to store the results,
-    # DataUnits stores by default MSR, phase, i, q
+    # DataUnits stores by default Signal, phase, i, q
     # additionally include wait time and t_max
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/t2_signal.py:67]: Delete

File: src/qibocal/protocols/characterization/coherence/t1_signal.py (hunk 2 above)
Comment (andrea-pasquale) [reply src/qibocal/protocols/characterization/coherence/t1_signal.py:177]: Yes, maybe "Signal [a.u.]". I think we should use square brackets everywhere.

Comment (GabrielePalazzo) [reply src/qibocal/protocols/characterization/coherence/t1_signal.py]: Done.

File: src/qibocal/protocols/characterization/coherence/spin_echo_signal.py [hunk 8]
```diff
@@ -159,17 +159,17 @@ def _plot(data: SpinEchoMSRData, qubit, fit: SpinEchoMSRResults = None):
         showlegend=True,
         uirevision="0",  # ``uirevision`` allows zooming while live plotting
         xaxis_title="Time (ns)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/spin_echo_signal.py:161]:
```suggestion
        xaxis_title="Time [ns]",
```

File: src/qibocal/protocols/characterization/coherence/t1_signal.py [hunk 9]
```diff
@@ -174,13 +173,13 @@ def _plot(data: T1MSRData, qubit, fit: T1MSRResults = None):
         showlegend=True,
         uirevision="0",  # ``uirevision`` allows zooming while live plotting
         xaxis_title="Time (ns)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/t1_signal.py:175]:
```suggestion
        xaxis_title="Time [ns]",
```

File: src/qibocal/protocols/characterization/coherence/t2_signal.py [hunk 10]
```diff
@@ -154,17 +151,17 @@ def _plot(data: T2MSRData, qubit, fit: T2MSRResults = None):
         showlegend=True,
         uirevision="0",  # ``uirevision`` allows zooming while live plotting
         xaxis_title="Time (ns)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/coherence/t2_signal.py:153]:
```suggestion
        xaxis_title="Time [ns]",
```

File: src/qibocal/protocols/characterization/dispersive_shift.py [hunk 11]
```diff
@@ -223,7 +222,7 @@ def _plot(data: DispersiveShiftData, qubit, fit: DispersiveShiftResults):
         horizontal_spacing=0.1,
         vertical_spacing=0.1,
         subplot_titles=(
-            "MSR (V)",
+            "Signal [a.u.]",
             "phase (rad)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/dispersive_shift.py:226]:
```suggestion
            "phase [rad]",
```

File: src/qibocal/protocols/characterization/dispersive_shift.py [hunk 12]
```diff
@@ -341,7 +340,7 @@ def _plot(data: DispersiveShiftData, qubit, fit: DispersiveShiftResults):
     fig.update_layout(
         showlegend=True,
         xaxis_title="Frequency (GHz)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/dispersive_shift.py:342]:
```suggestion
        xaxis_title="Frequency [GHz]",
```

File: src/qibocal/protocols/characterization/dispersive_shift.py [hunk 13]
```diff
@@ -341,7 +340,7 @@ def _plot(data: DispersiveShiftData, qubit, fit: DispersiveShiftResults):
...
-        yaxis_title="MSR (uV)",
+        yaxis_title="Signal [a.u.]",
         xaxis2_title="Frequency (GHz)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/dispersive_shift.py:344]:
```suggestion
        xaxis2_title="Frequency [GHz]",
```

File: src/qibocal/protocols/characterization/dispersive_shift.py [hunk 14]
```diff
@@ -341,7 +340,7 @@ def _plot(data: DispersiveShiftData, qubit, fit: DispersiveShiftResults):
...
         yaxis2_title="Phase (rad)",
```
Comment (Edoardo-Pedicillo) [inline src/qibocal/protocols/characterization/dispersive_shift.py:345]:
```suggestion
        yaxis2_title="Phase [rad]",
```

**Review by Edoardo-Pedicillo — COMMENTED** [review]
I have started to change some labels, but I realized later they are to many.

**Comment by GabrielePalazzo** [comment]
> I have started to change some labels, but I realized later they are to many :)

Yeah, I think we are falling out of scope for this PR. If we really want to get rid of parentheses for all units of measure we should open a separate pull request, otherwise it is going to take ages to refactor all of them...

**Comment by GabrielePalazzo** [comment]
The last commit also closes #614.

File: src/qibocal/protocols/characterization/coherence/t1_sequences.py [hunk 15]
```diff
@@ -50,8 +50,8 @@ def _acquisition(
         params.delay_before_readout_step,
     )
 
-    # create a DataUnits object to store the MSR, phase, i, q and the delay time
-    data = t1_msr.T1MSRData()
+    # create a DataUnits object to store the Signal, phase, i, q and the delay time
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/coherence/t1_sequences.py:53]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/coherence/t1_sequences.py [hunk 16]
```diff
@@ -70,19 +70,19 @@ def _acquisition(
             ),
         )
         for qubit in qubits:
-            # average msr, phase, i and q over the number of shots defined in the runcard
+            # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/coherence/t1_sequences.py:73]: Also this one

File: src/qibocal/protocols/characterization/couplers/coupler_qubit_spectroscopy.py [hunk 17]
```diff
@@ -106,12 +106,12 @@ def _acquisition(
     for i, pair in enumerate(qubits):
         # TODO: May measure both qubits on the pair
         qubit = platform.qubits[params.measured_qubits[i]].name
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/couplers/coupler_qubit_spectroscopy.py:109]: Remove outdated comment

File: src/qibocal/protocols/characterization/couplers/coupler_resonator_spectroscopy.py [hunk 18]
```diff
@@ -110,12 +110,12 @@ def _acquisition(
     for i, pair in enumerate(qubits):
         # TODO: May measure both qubits on the pair
         qubit = platform.qubits[params.measured_qubits[i]].name
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/couplers/coupler_resonator_spectroscopy.py:113]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/dispersive_shift.py [hunk 19]
```diff
@@ -151,7 +150,7 @@ def _acquisition(
 
     # retrieve the results for every qubit
     for qubit in qubits:
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/dispersive_shift.py:153]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/qubit_spectroscopy.py [hunk 20]
```diff
@@ -101,14 +101,14 @@ def _acquisition(
 
     # retrieve the results for every qubit
     for qubit, ro_pulse in ro_pulses.items():
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/qubit_spectroscopy.py:104]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/qubit_spectroscopy_ef.py [hunk 21]
```diff
@@ -137,14 +137,14 @@ def _acquisition(
 
     # retrieve the results for every qubit
     for qubit, ro_pulse in ro_pulses.items():
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/qubit_spectroscopy_ef.py:140]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/resonator_punchout_attenuation.py [hunk 22]
```diff
@@ -139,11 +131,11 @@ def _acquisition(
 
     # retrieve the results for every qubit
     for qubit in qubits:
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/resonator_punchout_attenuation.py:134]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/resonator_spectroscopy.py [hunk 23]
```diff
@@ -135,14 +135,14 @@ def _acquisition(
 
     # retrieve the results for every qubit
     for qubit in qubits:
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/resonator_spectroscopy.py:138]: This outdated comment can be removed.

File: src/qibocal/protocols/characterization/resonator_spectroscopy_attenuation.py [hunk 24]
```diff
@@ -126,14 +126,14 @@ def _acquisition(
 
     # retrieve the results for every qubit
     for qubit in qubits:
-        # average msr, phase, i and q over the number of shots defined in the runcard
+        # average signal, phase, i and q over the number of shots defined in the runcard
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/resonator_spectroscopy_attenuation.py:129]: This outdated comment can be removed.

**Review by andrea-pasquale — APPROVED** [review]
Thanks @GabrielePalazzo.
I still need to test on hardware but generally it looks good to me.
You will find a bunch of comments about removing outdated comments.
Make sure to open an issue regarding the refactor () -> [] if we are not planning to do it here.

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Pull request merged**